src/test/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/jobsservice/CreateJobTest.java

```diff
@@ -73,8 +73,9 @@ public void should_create_a_trigger_with_datetime_specified_by_caller() throws E
         Job job = validJob();
         jobsService.create(job, "some service name");
 
-        assertThat(triggerPassedToScheduler().getStartTime())
-            .isEqualTo(Date.from(job.trigger.startDateTime.toInstant()));
+        Trigger trigger = triggerPassedToScheduler();
+
+        assertThat(trigger.getStartTime()).isEqualTo(Date.from(job.trigger.startDateTime.toInstant()));
```

Comment:
Because JobsService retrieves the trigger from the converter (and is not responsible for its details), we should check if we passed the right (platform) trigger to the converter and if we used the converted trigger. Start date should be tested in the converter's test.

Reply:
this would not have appeared in PR. leftover of previous tests. i don't want to make converter accessible to public. it's purely for internal reasons to convert passed job (essentially trigger) and schedule it. the test is checking the trigger passed to scheduler. same as jobdata in other tests. let me roll back this leftover so it does not confuse any1.
the other tests are in converter unit test suite

Reply:
package-private or not, the converter is an independent piece of code that is not a part of JobsService and could we tested separately.